larq/quantizers.py

```diff
@@ -2,31 +2,26 @@
 quantized output and the pseudo-gradient method used for the backwards pass."""
 
 import tensorflow as tf
-from larq import utils
-
-
-def sign(x):
-    """A sign function that will never be zero"""
-    return tf.sign(tf.sign(x) + 0.1)
+import larq as lq
```

Comment:
I'd prefer to still keep the import like `from larq import utils, math` so users can import `from larq import quantizers` without needing to implicitly import every submodule from larq (due to the `import larq as lq` in this file). I think (though I am not sure) this is what most other libraries to as well.

On the other hand TensorFlow takes ages to import anyway so we definitely don't get any noticable import speed improvements by only importing `utils` and `math` here.